flink-connectors/flink-connector-aws-kinesis-data-streams/pom.xml

```diff
@@ -118,16 +109,6 @@ under the License.
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.22.1</version>
-				<configuration>
```

Comment:
I think some of these may need to be included for the `KinesaliteContainer` to not fail intermittently on the build test server. Did you have a reason in mind when removing? :-)